sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaConverter.scala

```diff
@@ -547,7 +551,7 @@ class SparkToParquetSchemaConverter(
               .as(LogicalTypeAnnotation.timestampType(true, TimeUnit.MILLIS)).named(field.name)
         }
 
-      case TimestampNTZType =>
+      case TimestampNTZType if SQLConf.get.timestampNTZSupportEnabled =>
```

Comment:
Shall we put `timestampNTZSupportEnabled` in the constructor list of `SparkToParquetSchemaConverter`?